src/Propel/Runtime/Adapter/MSSQL/MssqlPropelPDO.php

```diff
@@ -63,7 +63,7 @@ public function commit()
                 } else {
                     $return = self::exec('COMMIT TRANSACTION');
                     if ($this->useDebug) {
-                        $this->log('Commit transaction', null, __METHOD__);
+                        $this->log('Commit transaction');
```

Comment:
What do you think about 

```suggestion
                        $this->log('Commit transaction: ' . __METHOD__);
```

?

Reply:
Sure, go ahead and add those on top
I was just looking into clear violations of PHP usage (phpstan detected errors). I didn't look for semantic issues that could be improved.